nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -534,10 +534,6 @@ Use `services.pipewire.extraConfig` or `services.pipewire.configPackages` for Pi
   - to get previous behaviour of upstream defaults, set it to `null`
   - default value has changed from `[]` to `null`, in order to preserve default behaviour
 
-- `androidenv.androidPkgs_9_0` has been removed, and replaced with `androidenv.androidPkgs` for a more complete Android SDK including support for Android 9 and later.
-
-- `gtest` package has been updated past v1.13.0, which requires C++14 or higher.
```

Comment:
Looks like this was accidentally removed?

Reply:
That gtest line was actually added by #315233. I was in the process of cleaning it up and rebasing the commit when it was merged :no_mouth: